tests/test_convolved.py

```diff
@@ -177,28 +177,27 @@ def check(self, psfFwhm=0.5, flux=1000.0, forced=False):
             kronRadius = source.get("ext_photometryKron_KronFlux_radius")
 
         self.assertFalse(source.get(algName + "_flag"))  # algorithm succeeded
-        for ii, seeing in enumerate(algConfig.seeing):
-            deconvolve = seeing < psfFwhm/scale.asArcseconds()
-            self.assertTrue(source.get(algName + "_%d_deconv" % ii) == deconvolve)
-            if deconvolve:
-                # Not worth checking anything else
-                continue
+        originalSeeing = psfFwhm/scale.asArcseconds()
+        for ii, targetSeeing in enumerate(algConfig.seeing):
+            deconvolve = targetSeeing < originalSeeing
+            self.assertEqual(source.get(algName + "_%d_deconv" % ii), deconvolve)
+            seeing = targetSeeing if not deconvolve else originalSeeing
```

Comment:
Would it be more clear to have `seeing = originalSeeing if deconvolve else targetSeeing`?